Week3/homework/5-step3.js

```diff
@@ -1,2 +1,17 @@
 'use strict';
-// paste your freeCodeCamp solutions in here
+function multiplyAll(arr) {
+  var product = 1;
+  for (var i = 0; i < arr.length; i++) {
+    for (var j = 0; j < arr[i].length; j++) {
+      console.log(arr[i][j]);
+      product = product * arr[i][j];
+       console.log(product * arr[i][j]);
+    }
+  }
+  return product;
```

Comment:
The console.log statements inside the function body are not telling me much. I'm far more interested in the final product, but you don't console.log, e.g.:

```js
```